sao.js

```diff
@@ -10,10 +10,10 @@ const getSalts = async () => {
 
 module.exports = {
 	prompts: {
-		name: {
-			message: 'Project name',
+		packageName: {
+			message: 'Package name (lowercase without whitespace)',
 			default: ':folderName:',
```

Comment:
Misschien dom hoor, maar waarom is de default dan camelcase als hierboven staat dat het lowercase moet zijn?

Reply:
:folderName: is een variabele